include/vkd3d_shader.h

```diff
@@ -63,7 +63,9 @@ struct vkd3d_shader_meta
     bool replaced;
     bool uses_subgroup_size;
     bool uses_native_16bit_operations;
+    bool padding;
```

Comment:
could we turn the bools into a uint32 bitfield? Makes it easier to add stuff later if needed.

Reply:
I can fix that, yes.

Reply:
Fixed.